app/src/main/java/com/github/se/eventradar/model/repository/event/MockEventRepository.kt

```diff
@@ -56,7 +58,9 @@ class MockEventRepository : IEventRepository {
     }
   }
 
-  override suspend fun getEventsByIds(ids: List<String>): Resource<List<Event>> {
+  override suspend fun getEventsByIds(
+      ids: List<String>
+  ): Resource<List<Event>> { // TODO add filter for non expired?
```

Comment:
The getEventsByIds function is designed to fetch events based on provided IDs. Filtering out past events might limit its flexibility for future use cases. For instance, consider a scenario where we want to implement a feature in version 2 that allows users to view all events they have attended in the past. In this case, we would access the current user's `eventsAttendeeList` and call `getEventsByIds(ids: List<String>) `with these IDs to provide a comprehensive past events recap.